src/main/java/mate/jdbc/dao/DriverDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Driver;
+
+public interface DriverDao {
+    Driver create(Driver driver);
+
+    Driver get(Long id);
```

Comment:
```suggestion
    Optional<Driver> get(Long id);
```

we should return Optional from DAO layer, where entity could be null, and the entity itself from Service layer